bluefog/torch/optimizers.py

```diff
@@ -31,6 +31,23 @@ class CommunicationType(Enum):
     allreduce = "allreduce"
     empty = "empty"
 
+_warning_message_num_step_per_communication = (
+    "Unexpected behavior:\n"
+    "  After num_steps_per_communication times of forward computation `y=model(x)` are called,\n"
+    "  an optimizer step() function must be called.\n"
+    "  It does not matter how many step() functions are called in between.\n"
+    "  Please adjust num_step_per_communication to update model parameters locally.\n"
+    "  More information can be found in the FAQ page.\n"
```

Comment:
Either add a TODO or give the URL link to the index page of the document website?

Reply:
Done.